app/src/main/java/com/portfolio/tasky/usecases/EmailPatternValidator.kt

```diff
@@ -0,0 +1,5 @@
+package com.portfolio.tasky.usecases
+
+interface EmailPatternValidator {
+    fun isValidEmailPattern(email : CharSequence) : Boolean
```

Comment:
Why not leave it as String instead of CharSequence?

Reply:
Sry, didn't gave much thought to this.

Done